autogen/oai/cohere.py

```diff
@@ -369,34 +388,31 @@ def oai_messages_to_cohere_messages(
             new_message = {
                 "role": "CHATBOT",
                 "message": message["content"],
+                "tool_calls": [{"name": tool_call_.get("function",{}).get("name"), "parameters": eval(tool_call_.get("function",{}).get("arguments"))} for tool_call_ in message["tool_calls"]],
```

Comment:
During testing with a chess program this line failed for me. Are you able to test with the programs in my client testing repo?
https://github.com/marklysze/AutoGenClientTesting

This line caused an error (chess program):
https://github.com/marklysze/AutoGenClientTesting/blob/main/test_chess.py

```
Exception has occurred: SyntaxError
invalid syntax (<string>, line 0)
  File "/home/autogen/autogen/autogen/oai/cohere.py", line 390, in <listcomp>
    "tool_calls": [{"name": tool_call_.get("function",{}).get("name"), "parameters": eval(tool_call_.get("function",{}).get("arguments"))} for tool_call_ in message["tool_calls"]],
                                                                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/autogen/autogen/autogen/oai/cohere.py", line 390, in oai_messages_to_cohere_messages
    "tool_calls": [{"name": tool_call_.get("function",{}).get("name"), "parameters": eval(tool_call_.get("function",{}).get("arguments"))} for tool_call_ in message["tool_calls"]],
```

I think it would also be best not to use `eval`, can you consider an alternative?

Reply:
Hey, yeah sure, just updated! I ran against the test cases, and they seem to run fine on my end! :)

Reply:
Marking as resolved!